Detectors/ITSMFT/ITS/tracking/include/ITStracking/Tracker.h

```diff
@@ -106,35 +93,21 @@ class Tracker
   TrackerTraits* mTraits = nullptr; /// Observer pointer, not owned by this class
   TimeFrame* mTimeFrame = nullptr;  /// Observer pointer, not owned by this class
 
-  std::vector<MemoryParameters> mMemParams;
   std::vector<TrackingParameters> mTrkParams;
-
-  int mNThreads = 1;
-  bool mCUDA = false;
-  bool mApplySmoothing = false;
-  o2::base::PropagatorImpl<float>::MatCorrType mCorrType = o2::base::PropagatorImpl<float>::MatCorrType::USEMatCorrNONE;
-  float mBz = 5.f;
   std::uint32_t mTimeFrameCounter = 0;
   o2::gpu::GPUChainITS* mRecoChain = nullptr;
 
   unsigned int mNumberOfRuns{0};
 };
 
-inline void Tracker::setParameters(const std::vector<MemoryParameters>& memPars, const std::vector<TrackingParameters>& trkPars)
+inline void Tracker::setParameters(const std::vector<TrackingParameters>& trkPars)
 {
-  mMemParams = memPars;
   mTrkParams = trkPars;
 }
 
-inline float Tracker::getBz() const
-{
-  return mBz;
-}
-
-template <typename... T>
-void Tracker::initialiseTimeFrame(T&&... args)
+inline void Tracker::initialiseTimeFrame(int& iteration)
```

Comment:
idem

Reply:
Not as trivial as it seems, somehow this raises errors in template variadic parameters expansion for the wrapper `evaluateTask(...)`.

E.g:
```bash
home/mconcas/alice/sw/SOURCES/O2/fully-separate-traits/0/Detectors/ITSMFT/ITS/tracking/include/ITStracking/Tracker.h:91:9: note:   template argument deduction/substitution failed:
2022-07-23@16:45:59:DEBUG:O2:O2:0: /home/mconcas/alice/sw/SOURCES/O2/fully-separate-traits/0/Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx:53:103: note:   inconsistent parameter pack deduction with 'int' and 'int&'
```

Hence I decided not to touch too much.